src/main/java/org/jitsi/srtp/crypto/HmacSha1.java

```diff
@@ -33,16 +34,23 @@
      * @return a new <tt>org.bouncycastle.crypto.Mac</tt> instance which
      * implements a keyed-hash message authentication code (HMAC) with SHA-1
      */
-    public static Mac createMac()
+    public static org.bouncycastle.crypto.Mac createMac()
     {
         if (OpenSslWrapperLoader.isLoaded())
         {
             return new OpenSslHmac(OpenSslHmac.SHA1);
         }
         else
         {
-            // Fallback to BouncyCastle.
-            return new HMac(new SHA1Digest());
+            // Fallback to JCE.
+            try
+            {
+                return new MacAdapter(Mac.getInstance("HmacSHA1"));
```

Comment:
We use the bouncycastle type to get the instance here?

Reply:
No, `Mac` in this file is `javax.crypto.Mac`.  Might be cleaner if I explicitly qualified it?